assets/src/edit-story/components/library/panes/media/media3p/api/test/useMedia3pApiProvider.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+/**
+ * External dependencies
+ */
+import { act, renderHook } from '@testing-library/react-hooks';
+import { useMedia3pApiProvider } from '../index';
+import Media3pApiProvider from '../media3pApiProvider';
+
+jest.mock('../apiFetcher', () => ({
+  listMedia: () =>
+    Promise.resolve({
+      media: [
+        {
+          name: 'media/unsplash:1234',
+          provider: 'UNSPLASH',
+          imageUrls: [
+            {
+              imageName: 'full',
+              url: 'http://www.img.com',
+              width: 200,
+              height: 100,
+              mimeType: 'image/png',
+            },
+          ],
+          description: 'A cat',
+          type: 'IMAGE',
+          createTime: '1234',
+          updateTime: '5678',
+        },
+      ],
+      nextPageToken: 'lala',
+    }),
+}));
+
+describe('useMedia3pApiProvider', () => {
+  it('should properly call listMedia and map the results', async () => {
+    const wrapper = (params) => (
+      <Media3pApiProvider>{params.children}</Media3pApiProvider>
+    );
+    const { result } = renderHook(() => useMedia3pApiProvider(), { wrapper });
+
+    let listMediaResult;
+    await act(async () => {
```

Comment:
act() is not required here, as listMedia doesn't perform any useEffect() actions or update the component state.